framework/include/geomsearch/ElementPairLocator.h

```diff
@@ -70,12 +72,10 @@ class ElementPairLocator
   }
 
 protected:
-
-  const std::list<std::pair<const Elem*, const Elem*> > * _elem_pairs;
-
+  const ElementPairList * _elem_pairs;
   std::map<const Elem*, ElementPairInfo> _element_pair_info;
 
-private:
+public:
```

Comment:
Sorry for not being clear for the first time. This should be `protected` and if you need to access it, add an getter for it.